.env

```diff
@@ -1,2 +1,7 @@
-GOOGLE_CLIENT_ID=315341524768-hsmadcl15vijh8a9gv3cks0rp46r5ngb.apps.googleusercontent.com
-GOOGLE_CLIENT_SECRET=St2eSMGy1IcY63gRS9Cawm5p
\ No newline at end of file
+FIREBASE_API_KEY=AIzaSyBfetXIUjgUNRY42M3dtdr1ol6-dV6P5RY
+AUTH_DOMAIN=newsfeed-165004.firebaseapp.com
+DATABASE_URL=https://newsfeed-165004.firebaseio.com
+PROJECT_ID=newsfeed-165004
+STORAGE_BUCKET=newsfeed-165004.appspot.com
+MESSAGING_SENDER_ID=315341524768
+NEWS_API_KEY=213327409d384371851777e7c7f78dfe
```

Comment:
Sensitive data shouldn't be pushed to your repo. 
You can always set them up on whatever services needs them.
I suggest you ignore this .env file (gitignore) and explore how to add these keys to services that might need them.